fdbserver/ApplyMetadataMutation.cpp

```diff
@@ -93,7 +93,7 @@ class ApplyMetadataMutationsImpl {
 	    storageCache(&proxyCommitData_.storageCache), tag_popped(&proxyCommitData_.tag_popped),
 	    tssMapping(&proxyCommitData_.tssMapping), tLogGroupCollection(proxyCommitData_.tLogGroupCollection),
 	    initialCommit(initialCommit_), tagToServer(&proxyCommitData_.tagToServer),
-	    ssToStorageTeam(&proxyCommitData_.ssToStorageTeam), changedTeams(&proxyCommitData_.changedTeams) {
+	    ssToStorageTeam(&proxyCommitData_.ssToStorageTeam) /* unused , changedTeams(&proxyCommitData_.changedTeams) */ {
```

Comment:
Can you clean up this? I think you had another PR addressing this already.

Reply:
Done